scripts/EN/Harry

```diff
@@ -3842,68 +3842,68 @@ HandleInput(); // 0x4b06: 00000010
+ShowText((1358)"Perhaps it's because he doesn't want to disturb me?¥nW1Even the hospital director has not come to see me much lately.¥nW1I don't like the director very much, so I am honestly quite happy."); // 0x4b66: 0000100c
 HandleInput(); // 0x4b6c: 00000010
 PlaySecondSoundFx(0x19); // 0x4b70: 00001073
-ShowText((1359)"April 16 P.S. By some strange chance I learned a certain person is coming to visit me."); // 0x4b76: 0000100c
+ShowText((1359)"April 16 P.S. ¥nBy some strange chance W1I learned a certain person is coming to visit me."); // 0x4b76: 0000100c
 HandleInput(); // 0x4b7c: 00000010
 PlaySecondSoundFx(0x19); // 0x4b80: 00001073
 ShowText((1360)"April 17"); // 0x4b86: 0000100c
 HandleInput(); // 0x4b8c: 00000010
-ShowText((1361)"Everyone has been in a rush for some reason since before sunrise this morning. Something about a dead person coming back to life. Does that really happen?"); // 0x4b90: 0000100c
+ShowText((1361)"Everyone has been in a rush for some reason since before sunrise this morning.¥nW2Something about a dead person coming back to life.¥nW2Does that really happen?"); // 0x4b90: 0000100c
 HandleInput(); // 0x4b96: 00000010
-ShowText((1362)"It is a frightening story... And because of that my meal was a little late. The cold soup is disappointing."); // 0x4b9a: 0000100c
+ShowText((1362)"It is a frightening story…¥nW2And because of that my meal was a little late.¥nW2The cold soup is disappointing."); // 0x4b9a: 0000100c
 HandleInput(); // 0x4ba0: 00000010
-ShowText((1363)"Suddenly I remembered him. Maybe he has forgotten me, but I definitely remember him. Lately I think of him often. I want to see him."); // 0x4ba4: 0000100c
+ShowText((1363)"Suddenly I remembered him.¥nW2Maybe he has forgotten me, W1but I definitely remember him.¥nW2Lately I think of him often.¥nW2I want to see him."); // 0x4ba4: 0000100c
 HandleInput(); // 0x4baa: 00000010
-ShowText((1364)"It looks like my visitor is coming in the evening on the day after tomorrow. Before then..."); // 0x4bae: 0000100c
+ShowText((1364)"It looks like my visitor is coming in the evening on the day after tomorrow.¥nW2Before then…"); // 0x4bae: 0000100c
 HandleInput(); // 0x4bb4: 00000010
-ShowText((1365)"Before then just once."); // 0x4bb8: 0000100c
+ShowText((1365)"Before then W1just once."); // 0x4bb8: 0000100c
 HandleInput(); // 0x4bbe: 00000010
 PlaySecondSoundFx(0x19); // 0x4bc2: 00001073
 ShowText((1366)"April 18"); // 0x4bc8: 0000100c
 HandleInput(); // 0x4bce: 00000010
-ShowText((1367)"It is finally tomorrow. I will not tell anyone about this. If I tell anyone they will probably strap me to my bed."); // 0x4bd2: 0000100c
+ShowText((1367)"Tomorrow is the day.¥nW2I will not tell anyone about this.¥nW2If I tell anyone they will probably strap me to my bed."); // 0x4bd2: 0000100c
```

Comment:
The "tomorrow" in the first sentence does indeed refer to April 19th. (Confirmed this with native Japanese speakers.)

Made a slight modification to make this more clear.